design-documents/graph-ql/coverage/gift-wrapping.md

```diff
@@ -45,6 +45,11 @@ type SalesItemInterface {
     gift_message: GiftMessage @doc(description: "The entered gift message for the order item")
 }
 
+interface OrderItemInterface {
```

Comment:
Why do we have both now `SalesItemInterface` and `OrderItemInterface`? Should `SalesItemInterface` just be renamed to `OrderItemInterface`?

Reply:
I was renamed as part of customer orders schema changes.